query/src/storages/fuse/io/meta_readers.rs

```diff
@@ -22,6 +22,7 @@ use common_exception::ErrorCode;
 use common_exception::Result;
 use common_tracing::tracing::debug_span;
 use common_tracing::tracing::Instrument;
+use futures::io::BufReader;
```

Comment:
Why use `futures::io::BufReader` instead of `tokio::io::BufReader` here?

Reply:
Follow the #3938 description, `futures::io::BufReader` was used in block_reader.rs. Actually I also have confusion since `std::io:BufReader` was also used in databend.

Reply:
`InputStream` is `futures::AsyncRead + futures::Seek + ... ` , which does not fit with other two `BufReader`s (at least out of box)

FYR:

https://github.com/datafuselabs/databend/blob/b9070da6d34682363acfb03ce78329052b1f40ef/common/dal/src/data_accessor.rs#L30-L32